backend/src/test/resources/tests/aggregator/DURATION_SUM_AGGREGATOR/content.csv

```diff
@@ -5,3 +5,4 @@ pid,datum
 4,"2010-06-16/2010-06-18"
 5,"2011-03-02/2011-03-10"
 5,"2011-03-05/2011-03-13"
+6,"/2011-03-13"
```

Comment:
Kein Case für "2011-03-13/" ?

Reply:
würde sich ja ändern mit dem heutigen Datum

Reply:
😀 thats my point

Reply:
das würde sich auch bei der erstausführung testen, was es ja in den tests ist.